MSMetaEnhancer/libs/converters/web/BridgeDB.py

```diff
@@ -13,31 +13,42 @@ def __init__(self, session):
         # service URLs
         self.endpoints = {'BridgeDB': 'https://webservice.bridgedb.org/Human/xrefs/'}
 
-        self.codes = {'hmdbid': 'Ch', 'pubchemid': 'Cpc', 'chemspiderid': 'Cs', 'wikidataid': 'Wd', 'chebiid': 'Ce'}
+        self.codes = {'hmdbid': 'Ch', 'pubchemid': 'Cpc', 'chemspiderid': 'Cs', 'wikidataid': 'Wd', 'chebiid': 'Ce',
+                      'keggid': 'Ck'}
         self.identifiers = {'PubChem-compound': 'pubchemid', 'Chemspider': 'chemspiderid', 'ChEBI': 'chebiid',
-                            'HMDB': 'hmdbid', 'Wikidata': 'wikidataid'}
+                            'HMDB': 'hmdbid', 'Wikidata': 'wikidataid', 'KEGG Compound': 'keggid'}
 
         # generate top level methods defining allowed conversions
         conversions = [('hmdbid', 'pubchemid', 'from_hmdbid'),
                        ('hmdbid', 'chemspiderid', 'from_hmdbid'),
                        ('hmdbid', 'wikidataid', 'from_hmdbid'),
                        ('hmdbid', 'chebiid', 'from_hmdbid'),
+                       ('hmdbid', 'keggid', 'from_hmdbid'),
                        ('pubchemid', 'hmdbid', 'from_pubchemid'),
                        ('pubchemid', 'chemspiderid', 'from_pubchemid'),
                        ('pubchemid', 'wikidataid', 'from_pubchemid'),
                        ('pubchemid', 'chebiid', 'from_pubchemid'),
+                       ('pubchemid', 'keggid', 'from_pubchemid'),
                        ('chemspiderid', 'hmdbid', 'from_chemspiderid'),
                        ('chemspiderid', 'pubchemid', 'from_chemspiderid'),
                        ('chemspiderid', 'wikidataid', 'from_chemspiderid'),
                        ('chemspiderid', 'chebiid', 'from_chemspiderid'),
+                       ('chemspiderid', 'keggid', 'from_chemspiderid'),
                        ('wikidataid', 'hmdbid', 'from_wikidataid'),
                        ('wikidataid', 'pubchemid', 'from_wikidataid'),
                        ('wikidataid', 'chemspiderid', 'from_wikidataid'),
                        ('wikidataid', 'chebiid', 'from_wikidataid'),
+                       ('wikidataid', 'keggid', 'from_wikidataid'),
                        ('chebiid', 'hmdbid', 'from_chebiid'),
                        ('chebiid', 'pubchemid', 'from_chebiid'),
                        ('chebiid', 'chemspiderid', 'from_chebiid'),
-                       ('chebiid', 'wikidataid', 'from_chebiid')
+                       ('chebiid', 'wikidataid', 'from_chebiid'),
+                       ('chebiid', 'keggid', 'from_chebiid'),
+                       ('keggid', 'hmdbid', 'from_keggid'),
+                       ('keggid', 'pubchemid', 'from_keggid'),
+                       ('keggid', 'chemspiderid', 'from_keggid'),
+                       ('keggid', 'wikidataid', 'from_keggid'),
+                       ('keggid', 'chebiid', 'from_keggid'),
                        ]
```

Comment:
I think this could be also at some point made as all possible combinations between the 4 initial identifiers - otherwise all good!